src/it/scala/org/broadinstitute/dsde/rawls/openam/OpenAmClientService.scala

```diff
@@ -0,0 +1,56 @@
+package scala.org.broadinstitute.dsde.rawls.openam
+
+import java.io.File
+import java.util.concurrent.TimeUnit
+
+import akka.actor.Actor
+import akka.event.Logging
+import com.typesafe.config.ConfigFactory
+import spray.client.pipelining._
+import spray.http.{HttpEntity, MediaTypes}
+import spray.json._
+
+import scala.concurrent.Await
+import scala.concurrent.duration.{Duration, FiniteDuration}
+
+import OpenAmClientService.{OpenAmAuthRequest,OpenAmResponse}
+
+object OpenAmClientService {
+  case class OpenAmAuthRequest(username: String, password: String)
+  case class OpenAmResponse(tokenId: String, successUrl: String)
+
+  val conf = ConfigFactory.parseFile(new File("/etc/rawls.conf"))
```

Comment:
We should try to minimize the number of places where we specify the conf file. Maybe it could be passed in here somehow (e.g. from ```Boot``` if we're running the app, or ```WorkspaceSimulation``` if we're running an integration test).